src/modules/pipelines/README.md

```diff
@@ -0,0 +1,184 @@
+MCA (Sep 2022):
+    invoiceId, previousInvoiceId, billingAccountId, billingAccountName, billingProfileId, billingProfileName, invoiceSectionId, invoiceSectionName, resellerName, resellerMpnId, costCenter, billingPeriodEndDate, billingPeriodStartDate, servicePeriodEndDate, servicePeriodStartDate, date, serviceFamily, productOrderId, productOrderName, consumedService, meterId, meterName, meterCategory, meterSubCategory, meterRegion, ProductId, ProductName, SubscriptionId, subscriptionName, publisherType, publisherId, publisherName, resourceGroupName, ResourceId, resourceLocation, location, effectivePrice, quantity, unitOfMeasure, chargeType, billingCurrency, pricingCurrency, costInBillingCurrency, costInPricingCurrency, costInUsd, paygCostInBillingCurrency, paygCostInUsd, exchangeRatePricingToBilling, exchangeRateDate, isAzureCreditEligible, serviceInfo1, serviceInfo2, additionalInfo, tags, PayGPrice, frequency, term, reservationId, reservationName, pricingModel, unitPrice, costAllocationRuleName, benefitId, benefitName, provider
+CSP (Sep 2022):
+    invoiceId, billingAccountId, billingAccountName, previousInvoiceId, billingProfileId, billingProfileName, invoiceSectionId, invoiceSectionName, partnerTenantId, partnerName, resellerName, resellerMpnId, customerTenantId, customerName, costCenter, billingPeriodStartDate, billingPeriodEndDate, servicePeriodStartDate, servicePeriodEndDate, date, serviceFamily, productOrderId, productOrderName, consumedService, meterId, meterName, meterCategory, meterSubCategory, meterRegion, productId, product, subscriptionId, subscriptionName, publisherType, publisherId, publisherName, resourceGroupName, resourceId, resourceLocation, location, effectivePrice, quantity, unitOfMeasure, chargeType, billingCurrency, pricingCurrency, costInBillingCurrency, costInPricingCurrency, costInUsd, paygCostInBillingCurrency, paygCostInUsd, exchangeRatePricingToBilling, exchangeRateDate, isAzureCreditEligible, serviceInfo1, serviceInfo2, additionalInfo, tags, partnerEarnedCreditRate, partnerEarnedCreditApplied, payGPrice, frequency, term, reservationId, reservationName, pricingModel, unitPrice, costAllocationRuleName, provider, benefitId, benefitName
+MG (Aug 2022):
+    DepartmentName, AccountName, AccountOwnerId, SubscriptionGuid, SubscriptionName, ResourceGroup, ResourceLocation, AvailabilityZone, UsageDateTime, ProductName, MeterCategory, MeterSubcategory, MeterId, MeterName, MeterRegion, UnitOfMeasure, UsageQuantity, ResourceRate, PreTaxCost, CostCenter, ConsumedService, ResourceType, InstanceId, Tags, OfferId, AdditionalInfo, ServiceInfo1, ServiceInfo2, Currency
+-->
+
+> ℹ️ _The table assumes all fields are Pascal-cased (first letter of each word capitalized)._
+
+| Column                     | Type   | Value                                                                                                       | Notes                                                   |
```

Comment:
Would love to also document maximum length for strings and format for Date, those are important for schema definitions.